_sass/main.scss

```diff
@@ -26,21 +60,31 @@ $btn-border-radius-sm: 100px;
    }
 }
 
+// Text overlaping image
 .imgTxt {
    position: absolute;
    top: 30%;
+   left: 60px;
    color: #fff;
-   left: 8%;
    h1 {   
       font-size: 8rem;
    }
-   
    text-shadow: 2px 2px 5px black;
 }
 
+
+
+// .imgt {
+//    width: 100%;
+//    max-width: 210px;
+//    min-width: 100px;
+//    padding: 20px;
+// }
```

Comment:
If something has been commented out entirely, you should remove it unless you think you'll need it again in the future. Also remove the extra blank lines at the top.